api/rs/slint/android.rs

```diff
@@ -46,19 +46,35 @@
 //!
 //! ## Building and Deploying
 //!
+//! Building a Rust application requires the target toolchain to be installed. You can install it via *rustup*:
+//!
+//! ```sh
+//! rustup target add aarch64-linux-android
+//! ```
+//!
+//! Please ensure that you have the Android NDK and SDK installed and properly set up in your development environment for the above command to work as expected.
+//! For detailed instructions on how to set up the Android NDK and SDK, please refer to the [Android Developer's guide](https://developer.android.com/studio/projects/install-ndk).
+//! The following environment variables need to be set:
+//! * `ANDROID_HOME`: The directory in which your Android SDK is located. Usually `$HOME/Android/Sdk`.
+//! * `ANDROID_NDK_ROOT`: The directory in which your Android NDK is located. Usually `$HOME/Android/Sdk/ndk/${NDK_VERSION}`. ${NDK_VERSION} is the version of the NDK you have installed.
+//! * `JAVA_HOME`: The directory in which your Java compiler (`javac`) is located. This variable is optional if a proper `javac` is found in your `$PATH`. 
```

Comment:
Less words, only use "proper" if it has a clear meaning or there's an ambiguity.

```suggestion
//! * `JAVA_HOME`: The directory in which your Java compiler (`javac`) is located. This variable is optional if a `javac` is found in your `$PATH`. 
```